package.json

```diff
@@ -45,26 +44,22 @@
     "cross-env": "^5.2.1",
     "css-loader": "^0.23.0",
     "cypress": "^6.4.0",
-    "eslint": "^7.18.0",
-    "eslint-config-prettier": "^7.2.0",
-    "eslint-config-standard": "^16.0.2",
-    "eslint-plugin-chai-friendly": "^0.6.0",
-    "eslint-plugin-cypress": "^2.11.2",
-    "eslint-plugin-html": "^6.1.1",
-    "eslint-plugin-import": "^2.22.1",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^3.3.1",
-    "eslint-plugin-promise": "^4.2.1",
-    "eslint-plugin-vue": "^7.5.0",
-    "eslint-webpack-plugin": "^2.5.0",
+    "eslint": "^4.19.1",
+    "eslint-config-standard": "^10.2.1",
+    "eslint-loader": "^1.9.0",
```

Comment:
This package is deprecated, we should update it later. It will also need to migrate webpack to version >4 to use the eslint-webpack-plugin.